File: zadanie2/play-crud/app/models/Order.scala
package models

import play.api.libs.json.{Json, OFormat}

case class Order(id: Int,
                 createdAt: String,
                 customer: Int,
                 isPaid: Boolean,
                 paidAt: String,
                 totalOrderValue: Float,
                 coupon: Int
                )

object Order {
    implicit val orderFormat: OFormat[Order] = Json.format[Order]
}

File: zadanie2/angular-shop/src/app/_models/Order.ts
export interface Order {
    id: number;
    createdAt: string;
    customer: number;
    isPaid: boolean;
    paidAt: string;
    totalOrderValue: number;
    coupon: number;
}


export interface OrderItem {
    id: number;
    order: number;
    product: number;
}
